concourse/steps/scan_container_images.py

```diff
@@ -152,6 +153,32 @@ def mail_body(self):
 
         return ''.join(parts)
 
+    def pdf_report_attachments(self):
+        attachments = []
+
+        def report_filename(upload_result):
+            return upload_result.component.name.replace('.','_').replace('/', '_') + '-report.pdf'
```

Comment:
not opposed to doing that - but why replace `.`? the `/`-part, is of course obviously a good idea